test/e2e/node_reuse_test.go

```diff
@@ -509,7 +516,7 @@ func updateNodeReuse(nodeReuse bool, m3machineTemplateName string, clusterClient
 
 	// verify that nodereuse is true
 	Expect(clusterClient.Get(ctx, client.ObjectKey{Namespace: namespace, Name: m3machineTemplateName}, &m3machineTemplate)).To(Succeed())
-	Expect(m3machineTemplate.Spec.NodeReuse).To(BeTrue())
+	Expect(m3machineTemplate.Spec.NodeReuse).To(BeEquivalentTo(nodeReuse))
```

Comment:
what's the difference here, I guess `BeTrue` more relevant ?

Reply:
This function is also used in case when we set the value to false not just when we set it to true that is why this has been modified.